### PR TITLE
ITTNOTIFY TASKS refactoring in PCA and Cov

### PR DESCRIPTION
**Pull request by mchernov-intel**

### CODE REVIEW

File: cpp/daal/src/algorithms/covariance/oneapi/covariance_oneapi_impl.i [hunk 1]
```diff
@@ -153,7 +153,7 @@ services::Status updateDenseCrossProductAndSums(bool isNormalized, size_t nFeatu
             DAAL_CHECK_STATUS_VAR(status);
 
             {
-                DAAL_ITTNOTIFY_SCOPED_TASK(compute.updateCrossProductAndSums.gemmSums);
+                DAAL_ITTNOTIFY_SCOPED_TASK(covariance.compute_internal_function.updateCrossProductAndSums.gemmSums);
```
Comment (michael-smirnov) [inline cpp/daal/src/algorithms/covariance/oneapi/covariance_oneapi_impl.i:156]:
```suggestion
                DAAL_ITTNOTIFY_SCOPED_TASK(covariance.internal_function.updateCrossProductAndSums.gemmSums);
```

File: cpp/daal/src/algorithms/pca/oneapi/pca_dense_correlation_batch_kernel_ucapi_impl.i [hunk 2]
```diff
@@ -73,7 +73,7 @@ Status PCACorrelationKernelBatchUCAPI<algorithmFPType>::compute(bool isCorrelati
     cachekey.add(fptype_name);
 
     {
-        DAAL_ITTNOTIFY_SCOPED_TASK(compute.buildProgram);
+        DAAL_ITTNOTIFY_SCOPED_TASK(PCA.batch.compute.buildProgram);
```
Comment (michael-smirnov) [inline cpp/daal/src/algorithms/pca/oneapi/pca_dense_correlation_batch_kernel_ucapi_impl.i:76]: internal_function?

File: cpp/daal/src/algorithms/pca/oneapi/pca_dense_correlation_batch_kernel_ucapi_impl.i [hunk 3]
```diff
@@ -187,7 +187,7 @@ template <typename algorithmFPType>
 services::Status PCACorrelationKernelBatchUCAPI<algorithmFPType>::correlationFromCovarianceTable(uint32_t nObservations, NumericTable & covariance,
                                                                                                  const services::Buffer<algorithmFPType> & variances)
 {
-    DAAL_ITTNOTIFY_SCOPED_TASK(compute.correlationFromCovarianceTable);
+    DAAL_ITTNOTIFY_SCOPED_TASK(PCA.batch.compute_internal_function.correlationFromCovarianceTable);
```
Comment (michael-smirnov) [inline cpp/daal/src/algorithms/pca/oneapi/pca_dense_correlation_batch_kernel_ucapi_impl.i:190]:
```suggestion
    DAAL_ITTNOTIFY_SCOPED_TASK(PCA.batch.internal_function.correlationFromCovarianceTable);
```

File: cpp/daal/src/algorithms/pca/oneapi/pca_dense_correlation_batch_kernel_ucapi_impl.i [hunk 4]
```diff
@@ -213,7 +213,7 @@ services::Status PCACorrelationKernelBatchUCAPI<algorithmFPType>::calculateVaria
                                                                                      NumericTable & covariance,
                                                                                      const services::Buffer<algorithmFPType> & variances)
 {
-    DAAL_ITTNOTIFY_SCOPED_TASK(compute.calculateVariances);
+    DAAL_ITTNOTIFY_SCOPED_TASK(PCA.batch.compute_internal_function.calculateVariances);
```
Comment (michael-smirnov) [inline cpp/daal/src/algorithms/pca/oneapi/pca_dense_correlation_batch_kernel_ucapi_impl.i:216]:
```suggestion
    DAAL_ITTNOTIFY_SCOPED_TASK(PCA.batch.internal_function.calculateVariances);
```

**Comment by napetrov** [comment]
@Alexandr-Solovev - do  we need this PR?

**Comment by Alexandr-Solovev** [comment]
@napetrov I think not, if I remember, we are switching to another profiler. In addition, covariance and PCA are present in oneAPI interfaces with the new profiler.

**Pull request closed without merging**